brainframe/cli/os_utils.py

```diff
@@ -50,11 +51,14 @@ def is_root():
     return os.geteuid() == 0
 
 
-def give_brainframe_group_rw_access(paths: List[Path]):
-    paths_str = [str(p) for p in paths]
-
-    run(["chgrp", "-R", "brainframe"] + paths_str)
-    run(["chmod", "g+rwx"] + paths_str)
+def give_brainframe_group_rwx_access(path: Path):
+    # Make the path's group the brainframe group
+    os.chown(str(path), -1, BRAINFRAME_GROUP_ID)
+    # The owner and brainframe group should have full permissions. Others can
+    # read and execute only.
+    os.chmod(
+        str(path), stat.S_IRWXU | stat.S_IRWXG | stat.S_IROTH | stat.S_IXOTH
```

Comment:
I know you reverted it, but Pathlib also has a chmod command

Reply:
https://docs.python.org/3/library/pathlib.html#pathlib.Path.chmod

Reply:
Oh, good to know! It's still not recursive as far as I can tell unfortunately.